components/Choice/index.jsx

```diff
@@ -33,7 +34,10 @@ export default class Choice extends React.Component {
     onFocus: PropTypes.func,
     options: PropTypes.array.isRequired,
     optionsPerRow: PropTypes.oneOf(['3', '4', '5', '6', '7', 3, 4, 5, 6, 7, null]),
-    query: PropTypes.object,
+    query: PropTypes.oneOfType([
```

Comment:
I'm not sure that's necessary for such a small check. I think it makes sense for large complex objects such as intls. In this case it's easy to read, especially in the docs since we list all of the propTypes there and if we use a const we would have to change the docs to show this. I am also not sure this is within the scope of updating the npm deps.